sklearnex/__init__.py

```diff
@@ -13,27 +13,32 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-#===============================================================================
+# ===============================================================================
 
-from .dispatcher import patch_sklearn
-from .dispatcher import unpatch_sklearn
-from .dispatcher import get_patch_names
-from .dispatcher import get_patch_map
-from .dispatcher import sklearn_is_patched
-from ._config import get_config, set_config, config_context
+from . import utils
+from ._config import config_context, get_config, set_config
+from .dispatcher import (
+    get_patch_map,
+    get_patch_names,
+    is_patched_instance,
+    patch_sklearn,
+    sklearn_is_patched,
+    unpatch_sklearn,
+)
 
 __all__ = [
-    'cluster', 'config_context', 'basic_statistics', 'decomposition', 'ensemble',
-    'get_config', 'get_patch_names', 'linear_model', 'manifold', 'metrics',
-    'neighbors', 'patch_sklearn', 'set_config', 'svm', 'unpatch_sklearn',
-    'utils', 'sklearn_is_patchedget_patch_map'
+    "utils",
+    "config_context",
+    "get_config",
+    "set_config",
+    "get_patch_map",
+    "get_patch_names",
+    "is_patched_instance",
+    "patch_sklearn",
+    "sklearn_is_patched",
+    "unpatch_sklearn",

```

Comment:
Submodules (cluster, model_selection, etc) should not be removed from `__all__` so module behavior is aligned with [sklearn](https://github.com/scikit-learn/scikit-learn/blob/main/sklearn/__init__.py#L85) and submodules can be imported through `*` symbol (`from sklearn[ex] import *`)